packages/nuxt-locokit/package.json

```diff
@@ -22,21 +22,30 @@
     "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground"
   },
   "dependencies": {
+    "@locokit/designsystem": "^0.8.0",
+    "@locokit/i18n": "0.8.0",
     "@locokit/sdk": "^0.8.0",
     "@nuxt/kit": "^3.0.0-rc.11",
     "@pinia/nuxt": "^0.4.2",
-    "pinia": "^2.0.22",
-    "primevue": "^3.16.2",
-    "tailwindcss": "^3.1.8"
+    "@vee-validate/rules": "^4.7.2",
+    "pinia": "^2.0.23",
+    "primevue": "^3.17.0",
+    "tailwindcss": "^3.1.8",
+    "vee-validate": "^4.7.2",
+    "vue-i18n": "^9.3.0-beta.6"
   },
   "devDependencies": {
-    "@locokit/designsystem": "^0.8.0",
     "@nuxt/module-builder": "latest",
-    "@nuxtjs/eslint-config-typescript": "latest",
+    "@nuxtjs/eslint-config-typescript": "^11.0.0",
     "@nuxtjs/tailwindcss": "^5.3.2",
-    "eslint": "latest",
+    "@typescript-eslint/eslint-plugin": "^5.40.0",
+    "@typescript-eslint/parser": "^5.40.0",
+    "@vue/eslint-config-typescript": "^11.0.2",
+    "eslint": "^8.25.0",
+    "eslint-plugin-nuxt": "^3.2.0",
+    "eslint-plugin-vue": "^9.6.0",
     "nuxt": "^3.0.0-rc.11",
-    "sass": "^1.54.9",
-    "vue-i18n": "^9.2.2"
+    "prettier": "^2.7.1",
```

Comment:
This package must not be here, we need to use the root prettier package.
Same for TS / eslint packages ?